### ISSUE TITLE
[ic-data-dable] Screen reader sorting indicator shows on screen when using RTL language

### ISSUE THREAD

**Issue by irl** [mi6/ic-ui-kit#2905]
## Summary of the bug 

When using a RTL language, (dir="rtl" on the <html>), the text "table unsorted" appears way over to the right instead of being off screen. See the screenshot below for an image of a live example of this.

## 🪜 How to reproduce 

1. Set dir="rtl" on <html>
2. Add an ic-data-table
3. Observe the text "table unsorted" appear when you scroll far enough to the right

## 📸 Screenshots or code 

![Uploading Screen Shot 2024-12-16 at 12.58.38.png…]()

## 🖥 📱 Device 
- Type: desktop
- Device: mac os
- Browser version: Firefox 133

## 🧐 Expected behaviour 

The text should be hidden, except for screen readers

## 📝 Acceptance Criteria  
If relevant, describe in full detail the different interactions and edge cases that the component or patterns needs to fulfil. 

*Given* <html> dir attribute is "rtl"
*When* an ic-data-table is used
*Then* the screen reader hint is not visible

## Additional info 

In testing, switching the CSS to use `right:` instead of `left:` fixed the issue.

The [:dir](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) selector could be used to fix this.

**Comment by irl** [comment]
There are numerous occasions of this error in the set of web-components, so it would be best to wait for a decision in #2906 before implementing a fix for the canary components.

**Comment by GCHQ-Developer-299** [comment]
Akin to #2906, pushed to future release as we don't official support RTL languages - however, if external contributor addresses issue beforehand, that's great

**Comment by GCHQ-Developer-530** [comment]
Might be useful: https://github.com/mi6/ic-ui-kit/pull/3264
